src/pages/workspace/tags/TagSettingsPage.tsx

```diff
@@ -52,6 +53,12 @@ function TagSettingsPage({route, navigation}: TagSettingsPageProps) {
         navigation.setParams({tagName: currentPolicyTag?.name});
     }, [tagName, currentPolicyTag, navigation]);
 
+    const tagApprover = PolicyUtils.getTagApproverRule(policyID, route.params?.tagName)?.approver ?? '';
+    const approverText = useMemo(() => {
+        const approver = PersonalDetailsUtils.getPersonalDetailByEmail(tagApprover);
```

Comment:
This method should be lightweight right? Do we still need to use the useMemo hook here?